components/src/dynamo/planner/environment/base.py

```
@@ -0,0 +1,247 @@
+    def _refresh_component_worker_info(
+        self, sub_type: SubComponentType, component_state, get_worker_info
+    ) -> None:
+        if (
+            component_state.info is not None
+            and component_state.info.max_num_batched_tokens is not None
+        ):
+            return
+
+        try:
+            fresh = get_worker_info(sub_type, self.config.backend)
+        except Exception as exc:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)